### DISCUSSION TITLE
Does -P --plugins argument get used?

### DISCUSSION THREAD

**Discussion by M15terHyde** [mitre/caldera#2697] [General]
https://github.com/mitre/caldera/blob/b6556777ec8bd287f124f935f74696bd7f03a858/server.py#L108

Looking at the args that can be passed to server.py All of them get used or referenced in some way except for -P --plugins.
Does it do anything? Get used?

**Comment by clenk** [comment]
It's used here: https://github.com/mitre/caldera/blob/695895b3961b3dc16c28b9e264af570d034cfe7d/server.py#L61
The default value causes it to load the all the plugins from the `caldera/plugins` dir. If you point it to a different dir, it should only load that plugin and none of the others. Or you could use it if you want to store plugins outside of the `caldera` dir for some reason.
